libp2p/Host.cpp

```diff
@@ -109,6 +110,10 @@ Host::~Host()
 void Host::start()
 {
     DEV_TIMED_FUNCTION_ABOVE(500);
+    auto nodeTable = this->nodeTable();
```

Comment:
Have you revisited this check? The `->nodeTable()` does a lock and returns a shared pointer.

Reply:
yes I fixed it in the other PR so it will be fixed here too when I rebase after merging the other one